assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -82,12 +82,9 @@ systemd:
         ExecStartPre=/usr/bin/bash -c "grep 'certificate-authority-data' /etc/kubernetes/kubeconfig | awk '{print $2}' | base64 -d > /etc/kubernetes/ca.crt"
         ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/cache/kubelet-pod.uuid
         ExecStartPre=/etc/kubernetes/configure-kubelet-cgroup-driver
-        # TODO: Workaround until https://github.com/coreos/afterburn/pull/358
-        # makes it into Flatcar. Then we can read COREOS_PACKET_IPV4_PRIVATE_GATEWAY_0
-        # from /run/metadata/flatcar and disable the template conditionals below.
         ExecStart=/bin/sh -c \
             "%{~ if bgp_node_labels != "" ~}
-            BGP_PEER_ADDRESS=$(ip route | grep '10.0.0.0/8' | awk {'print $3'}); \
+            BGP_PEER_ADDRESS=$(curl -s https://metadata.packet.net/metadata | jq -r .bgp_neighbors[0].peer_ips[0]); \
```

Comment:
Why the first one of the peers?

Reply:
Because MetalLB doesn't support multiple node peers. I've updated the commit message.

Reply:
Oh, right, I forgot about that :). Can you create an issue to do this in the future, then? So we don't forget :)

Reply:
https://github.com/kinvolk/lokomotive/issues/1014